lib/fog/azurerm/models/network/network_interface.rb

```diff
@@ -25,24 +25,31 @@ class NetworkInterface < Fog::Model
 
         def self.parse(nic)
           nic_properties = nic['properties']
-          nic_ip_configuration = nic['properties']['ipConfigurations'][0]
+          nic_ip_configuration = nic_properties['ipConfigurations'][0]
           hash = {}
           hash['id'] = nic['id']
           hash['name'] = nic['name']
           hash['location'] = nic['location']
           hash['resource_group'] = nic['id'].split('/')[4]
```

Comment:
Instead of using magic numbers, use common method that would return resource_group from id. This is just FYI  point. No action needed at the moment